lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -496,6 +496,39 @@ static int process_cmd(const tunnel_comand *cmd, command_cb cb, void *ctx) {
             return 0;
         }
 
+        case TunnelCommand_RemoveIdentity: {
+            tunnel_delete_identity delete_id = {0};
+            if (cmd->data == NULL || parse_tunnel_delete_identity(&delete_id, cmd->data, strlen(cmd->data)) != 0) {
+                result.success = false;
+                result.error = "invalid command";
+                free_tunnel_delete_identity(&delete_id);
+                break;
+            }
+            struct ziti_instance_s *inst = model_map_get(&instances, delete_id.identifier);
+
+            if (inst == NULL) {
+                result.error = "ziti context not found";
+                result.success = false;
+                free_tunnel_delete_identity(&delete_id);
+                break;
+            }
+            if (inst) {
```

Comment:
At this point we can be sure that `inst` is not NULL, because the `break` on line 513 would prevent this line from being reached if `inst` was NULL. So at the very least this test on `inst` is not needed here.